.azure/app-cloud-e2e.yml

```diff
@@ -0,0 +1,144 @@
+      clean: all
+    steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versi`onSpec: '3.8'
+
+    - script: |
+        ID=$(date +%s)
+        echo "##vso[task.setvariable variable=id;isOutput=true]$ID"
+      # TODO: parse the PR number
+      name: local
+
+    - task: Cache@2
+      inputs:
+        key: 'pip | "$(name)" | requirements/app/base.txt'
+        restoreKeys: |
+          pip | "$(Agent.OS)"
+        path: $(pip_cache_dir)
+      displayName: Cache pip
+
+    - bash: |
+        python --version
+        pip --version
+      displayName: 'Info'
+
+    - bash: python -m pip install -r requirements/app/devel.txt --quiet --find-links ${TORCH_URL}
+      env:
+        TORCH_URL: https://download.pytorch.org/whl/cpu/torch_stable.html
+      displayName: 'Install dependencies'
+
+    - bash: |
+        python -m pip install playwright
+        python -m playwright install --with-deps
+      displayName: 'Install Playwright system dependencies'
+
+    - bash: pip install -e .
+      displayName: 'Install lightning'
+
+    - bash: |
+        git clone https://github.com/Lightning-AI/LAI-lightning-template-jupyterlab-App examples/app_template_jupyterlab
+        cp examples/app_template_jupyterlab/tests/test_template_jupyterlab.py tests/tests_app_examples/test_template_jupyterlab.py
+      condition: eq(variables['name'], 'template_jupyterlab')
+      displayName: 'Clone Template Jupyter Lab Repo'
+
+    - bash: git clone https://github.com/Lightning-AI/lightning-template-react examples/app_template_react_ui
+      condition: eq(variables['name'], 'template_react_ui')
+      displayName: 'Clone Template React UI Repo'
+
+    - bash: |
+        mkdir -p ${VIDEO_LOCATION}
+        ls -l examples/${TEST_APP_NAME}
+        ls -l tests/tests_app_examples
+        python -m pytest tests/tests_app_examples/test_${TEST_APP_NAME}.py::test_${TEST_APP_NAME}_example_cloud --timeout=900 --capture=no -v --color=yes
+      env:
+        HEADLESS: "1"
+        PACKAGE_LIGHTNING: "1"
+        CLOUD: "1"
+        VIDEO_LOCATION: '$(Build.ArtifactStagingDirectory)/videos'
+        PR_NUMBER: $(variables['local.id'])
+        TEST_APP_NAME: $(name)
+        HAR_LOCATION: ./artifacts/hars
+        SLOW_MO: 50
+        LAI_USER: $(LAI_USER)
+        LAI_PASS: $(LAI_PASS)
+        LIGHTNING_USER_ID: $(LIGHTNING_USER_ID)
+        LIGHTNING_API_KEY: $(LIGHTNING_API_KEY)
+        LIGHTNING_USERNAME: $(LIGHTNING_USERNAME)
+        LIGHTNING_CLOUD_URL: $(LIGHTNING_CLOUD_URL)
+      displayName: 'Run the tests'
+
+    - publish: '$(Build.ArtifactStagingDirectory)/videos'
+      displayName: 'Publish videos'
+      artifact: $(name)
+
+    - bash: time python -c "from lightning.app import testing; testing.delete_cloud_lightning_apps()"
+      env:
+        LAI_USER: $(LAI_USER)
+        LAI_PASS: $(LAI_PASS)
+        LIGHTNING_USER_ID: $(LIGHTNING_USER_ID)
+        LIGHTNING_API_KEY: $(LIGHTNING_API_KEY)
+        LIGHTNING_USERNAME: $(LIGHTNING_USERNAME)
```

Comment:
Can you double check if the `LIGHTNING_USERNAME` is right? The tests that fetches the log is failing and is likely because of the USERNAME

Reply:
ping me with correct one :)